src/platform/OpenThread/GenericThreadStackManagerImpl_OpenThread.cpp

```diff
@@ -469,6 +469,11 @@ ConnectivityManager::ThreadDeviceType GenericThreadStackManagerImpl_OpenThread<I
     if (linkMode.mRxOnWhenIdle)
         ExitNow(deviceType = ConnectivityManager::kThreadDeviceType_MinimalEndDevice);
 
+#if OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE
```

Comment:
I think this is an internal OpenThread config and doesn't need to be known here. We should rather rely on `CHIP_DEVICE_CONFIG_THREAD_SSED` only.